doc/collections/_pages/content/setup--windows/chapter-0/Readme.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Connecting to the Interwebs (Wireless Networking)
+---
+
+<!-- Begin GitHub-Flavored Markdown (GFM)
+See: https://docs.github.com/get-started/writing-on-github
+Spec: https://github.github.com/gfm/
+-->
+
+Before we can begin using our Windows Server machine as an effective software
+development environment, there are a few hurdles to overcome. This chapter deals
+with enabling and configuring connectivity to the Internet.
+
+```
+reg add HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\RadioManagement\SystemRadioState /ve /t REG_DWORD /d 0 /f
+```
+
+## Access to WiFi
+
+By default, access to WiFi is disabled in Windows Server 2022. This is because Airplane mode is enabled.
+It is also in an unconfigurable state (this preference cannot be changed by default). To be able to modify this
+preference, the following command must be run in an elevated Command Line Shell prompt.
```

Comment:
```suggestion
As a safe default, on fresh installations of Windows Server, WiFi connectivity
is inaccessible because Airplane mode is enabled.

Airplane mode is also unconfigurable (the default preference of "enabled" cannot
be changed). The following command must be run in an elevated Command Line Shell
prompt to modify this preference.
```

Reply:
I think there may actually be two separate safe defaults that need to be overridden here and a step may be missing

Reply:
https://www.windowscentral.com/how-enable-or-disable-airplane-mode-windows-11#command-line-method-6

Reply:
> you may not be able to turn it on or off, or the feature may appear grayed out.

Reply:
https://www.windowscentral.com/how-enable-or-disable-airplane-mode-windows-11#command-line-method-5

Should we document creating a restore point too? Seems smart